### DISCUSSION TITLE
2m-2s-asyn cluster mode When 2m is down

### DISCUSSION THREAD

**Discussion by hakusai22** [apache/rocketmq#8863] [Q&A]
- 2m-2s-asyn 集群模式 当2master down掉的时候，两个salve能不能提供生产写消息和消费读消息的功能？ 我看资料说只能提供读的功能,没有说到写的功能？

- 目前我测试完把2master杀掉之后，发现mq不能正常工作了，出现异常：No route info of this topic: xxxx。

**Comment by able0093** [comment]
从节点应该是只能把剩余消息消费完

**Reply by able0093** [reply]
建议部署Controller,可以切换

**Reply by xhgushen** [reply]
> 从节点应该是只能把剩余消息消费完

从节点也不能消费，是传统的1主1从，主挂了，重也消费不了
![Uploading 1769949592892.png…]()
